### PR TITLE
Global exception handler work

### PR DESCRIPTION
**Pull request by qreqit**

### CODE REVIEW

File: src/main/java/mate/academy/springbootwebgreqit/model/Book.java [hunk 1]
```diff
@@ -17,17 +19,24 @@ public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @Column(nullable = false, length = 100)
+    @Column(length = 100)
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/springbootwebgreqit/model/Book.java:22]: Left only 
```
@Column(nullable = false, length = 100)
```

same for others

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see pr

File: src/main/java/mate/academy/springbootwebgreqit/model/Book.java (hunk 1 above)
Comment (qreqit) [reply src/main/java/mate/academy/springbootwebgreqit/model/Book.java:22]: I wanted to use validation with @NotNull there, but ok)

**Review by sarakhmen — APPROVED** [review]
lgtm!

**Pull request merged**